Scenario Based Samples/Dynamic connection string sample/README.md

```diff
@@ -0,0 +1,130 @@
+
+ 2. Open the `ConnectionController.cs` file and enter the user1 details by referring to the image below.
+    * Enter the identity value, user1 email, and full name in CustomIdentity1.
+    * Enter SQL Server Id in the DataSource. (Eg: `DataSource = "SYNCLAPN-12345"`)
+    * Enter the Database Name in the InitialCatalog. (Eg: `InitialCatalog = "DCS_DB1"`)
+    * Enter the SQL Server Username and Password.
+
+    ![DCS_sample_user1](https://github.com/boldbi/samples/assets/129487075/d00edb1a-f80c-4d05-9338-52248ede7b45)
+
+ 3. Enter the user2 details by referring to the image below.
+    * Enter the identity value, user2 email and full name in CustomIdentity2.
+    * Enter SQL Server Id in the DataSource. (Eg: `DataSource = "SYNCLAPN-12345"`)
+    * Enter the Database Name in the InitialCatalog. (Eg: `InitialCatalog = "DCS_DB2"`)
+    * Enter the SQL Server Username and Password.
+
+    ![DCS_sample_user2](https://github.com/boldbi/samples/assets/129487075/c1536d19-1c97-48f6-a676-8916c6ae06c0)
+
+ 4. Run your sample in Visual Studio. The sample will render with the localhost URL http://localhost:63139.  Then add `/api/get-connection-details` to the URL. For example,  http://localhost:63139/api/get-connection-details.
+
+    ![DCSsampleAPI](https://github.com/boldbi/samples/assets/129487075/54c6f23c-35cc-4dc4-8630-b2d4e1241e2f)
+
+ 5. In BoldBI Server, create a SQL Server datasource with the first database (Eg: DCS_DB1). Enable Dynamic Connection String and click the `Configure` button. The image attached below for your reference will appear. Enter the localhost URL in the designated URL field.
+
+    ![dynamic-first](https://github.com/boldbi/samples/assets/129487075/53d12536-765d-49bf-83d9-ee724708eb74)
+
+ 6. ​​​​​​​In Configuration mode, if you are examining the dynamic connection string sample at the embedding level, choose `Embedding`. If at the server level, choose `Server`, or if both, choose the `Both` radio button. I have selected the `Both` radio button to test the sample at both server and embedding levels. In the User Identity choose `Email` or `Full Name` and connect it.​​​​​​​
+
+    > **NOTE:** At the embedding level, the identityType always holds the value as `Custom` in the Dynamic Connection String sample, even if you choose `Email` or `Full Name`. At the server level, it is based on the selection made in User Identity.
+
+     ![dynamic-second1](https://github.com/boldbi/samples/assets/129487075/5f3e542d-f5c2-4241-94de-aebe65050797)
+
+ 7. Create a dashboard with this same data source and publish it.
+
+## Steps to configure the Dynamic Connection String dashboard in the JS-based ASP.NET Core Embedding sample
+
+ 1. Please refer this [link](https://github.com/boldbi/aspnet-core-sample) to run the ASP.NET Core sample. You will need to render the sample with the created dashboard for Dynamic Connection String.
+
+ 2. Set the identity in the dynamicConnection member in the `Index.js` file by referring to the code below.
```

Comment:
Enable the isEnabled property and set the identity in the dynamicConnection member

Reply:
updated content